sensitivity-analysis/src/test/java/com/farao_community/farao/sensitivity_analysis/SystematicSensitivityResultTest.java

```diff
@@ -123,7 +135,7 @@ public void testPtdfResultManipulation() {
     public void testNokSensiResult() {
```

Comment:
how come this test still tests "not ok" result ?

Reply:
renamed the test

Now the SensitivityResult's default status is FAILURE, and if we find at least one NaN value in the powsybl result, we set the status to SUCCESS. This test is to make sure the status is indeed FAILURE when there are no values.